### PR TITLE
Make stop time update and array in TrioUpdate type

### PR DESCRIPTION
**Pull request by magnusburton**

### CODE REVIEW

**Comment by k-yle** [comment]
thanks for the PR, do you have a link to a feed where the data is in this format?

**Comment by magnusburton** [comment]
Hi! I'm working with the feed from [Trafiklab in Sweden](https://www.trafiklab.se/api/trafiklab-apis/gtfs-regional/realtime-specification/#/GTFS-RT/fetchTripUpdates), but the multiple stop time updates are also shown over at [Google](https://developers.google.com/transit/gtfs-realtime/examples/trip-updates-full).
Best,

File: src/realtime.def.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ export interface Vehicle {
 
 export interface TripUpdate {
   trip: RealTimeTrip;
-  stop_time_update?: StopTimeUpdate;
+  stop_time_update?: StopTimeUpdate[];
```
Comment (magnusburton) [inline src/realtime.def.ts:58]: Might want to rename this to `stop_time_updates` but that'd be a breaking change.

Comment (k-yle) [reply src/realtime.def.ts:58]: We don't choose the field names, they're defined in [the GTFS spec](https://gtfs.org/realtime/reference/#message-tripupdate)

**Comment by k-yle** [comment]
The reason that it's not an array is because the official feed for Auckland is not spec compliant, which I only just realised... 

In protobuf terminology, this field is `repeated`, so it should be defined as an array. I think we can release this PR as v4

**Pull request merged**